djangosaml2/backends.py

```diff
@@ -72,6 +72,7 @@ def authenticate(self, session_info=None, attribute_mapping=None,
             logger.error('Session info or attribute mapping are None')
             return None
 
+        logger.debug('session_info: {}'.format(session_info))
```

Comment:
Watch out for such use of logging! These logs can't be easily aggregated and processed. Always pass logging variables as separate arguments to the logger function.